lib/utils/validateRequired.js

```diff
@@ -35,3 +35,19 @@ export function validateRequiredPositiveAmount(value) {
     ? undefined
     : <FormattedMessage id="stripes-acq-components.validation.shouldBePositiveAmount" />;
 }
+
+export function validateRequiredMinAndMaxNumber({ minNumber, maxNumber, value }) {
+  if (value === undefined || value === null || value === '') {
```

Comment:
Can we simplify it to 
```suggestion
  if (!value) {
```

?

Reply:
I can do `!value && value !== 0` since `0` is false. Otherwise, we will receive `Required!` instead of `Value must be greater than or equal to 1`